opm/input/eclipse/EclipseState/Grid/FieldProps.cpp

```diff
@@ -672,16 +673,62 @@ bool FieldProps::supported<int>(const std::string& keyword) {
+
+    for(const auto& [mult_keyword, kw_info]: multiplier_kw_infos_)
+    {
+        const std::string keyword = mult_keyword.substr(prefix.size());
+        auto mult_iter = this->double_data.find(mult_keyword);
+        assert(mult_iter != this->double_data.end());
+        auto iter = this->double_data.find(keyword);
+        if (iter == this->double_data.end()) {
+            iter = this->double_data
+                .emplace(std::piecewise_construct,
+                         std::forward_as_tuple(keyword),
+                         std::forward_as_tuple(kw_info, this->active_size, kw_info.global ? this->global_size : 0))
+                .first;
+        }
+        using Scalar = typename std::remove_cv_t<std::remove_reference_t<decltype(iter->second.data[0])>>;
+        std::transform(iter->second.data.begin(), iter->second.data.end(),
+                       mult_iter->second.data.begin(), iter->second.data.begin(),
+                       std::multiplies<Scalar>());
+
+        // If data is global, then we also need to set the global_data. I think they should be the same at this stage, though!
+        if (kw_info.global)
+        {
+            assert(mult_iter->second.global_data.has_value() && iter->second.global_data.has_value());
+            std::transform(iter->second.global_data->begin(),
+                           iter->second.global_data->end(),
+                           mult_iter->second.global_data->begin(),
+                           iter->second.global_data->begin(),
+                           std::multiplies<Scalar>());
+        }
+        this->double_data.erase(mult_iter);
+    }
+    multiplier_kw_infos_.clear();
+}
 
 template <>
-Fieldprops::FieldData<double>& FieldProps::init_get(const std::string& keyword_name, const Fieldprops::keywords::keyword_info<double>& kw_info) {
+Fieldprops::FieldData<double>& FieldProps::init_get(const std::string& keyword_name, const Fieldprops::keywords::keyword_info<double>& kw_info,
+                                                    bool multiplier_in_edit) {
+    
+    if(multiplier_in_edit && !kw_info.scalar_init.has_value())
+        OPM_THROW(std::logic_error, std::string("Keyword " +  keyword_name + " is a multiplier and should have a default initial value."));
     const std::string& keyword = Fieldprops::keywords::get_keyword_from_alias(keyword_name);
+    const std::string& mult_keyword = std::string(multiplier_in_edit? getMultiplierPrefix() : "") + keyword;
 
-    auto iter = this->double_data.find(keyword);
-    if (iter != this->double_data.end())
+    auto iter = this->double_data.find(mult_keyword);
+    if (iter != this->double_data.end()) {
         return iter->second;
+    } else if(multiplier_in_edit){
+        assert(keyword != ParserKeywords::PORV::keywordName && keyword != ParserKeywords::TEMPI::keywordName &&
+               (Fieldprops::keywords::PROPS::satfunc.count(keyword) != 1) && !is_capillary_pressure(keyword));
```

Comment:
Okay, now I get it&ndash;this makes a lot more sense.  Thanks a lot for the update!  I'll merge into master.